android/hybridcommon/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -538,12 +540,13 @@ fun configure(
     context: Context,
     apiKey: String,
     appUserID: String?,
-    purchasesAreCompletedBy: PurchasesAreCompletedBy? = null,
+    purchasesAreCompletedBy: String? = null,
```

Comment:
I changed the APIs here and iOS to be strings instead of the enum itself. The reason is that most hybrids require us to pass something from the hybrid layer to the native (a string right now) and then we need to convert that string to the enum in each hybrid. I think it's better to just pass the string and perform the conversion at the PHC level. Lmk if you think otherwise!

Reply:
I think that makes sense! It's also consistent with other enums, like `verificationMode`.